src/hotspot/share/gc/shenandoah/shenandoahRootProcessor.cpp

```diff
@@ -193,7 +193,6 @@ void ShenandoahConcurrentRootScanner::roots_do(OopClosure* oops, uint worker_id)
   _vm_roots.oops_do(oops, worker_id);
 
   if (!heap->unload_classes()) {
-    AlwaysTrueClosure always_true;
```

Comment:
This change seems unrelated?

Reply:
Right. Restored and will file a separate bug to clean it up.